src/screens/ResultScreen.js

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from 'react';
+import {
+  View,
+  Text,
+  TouchableOpacity,
+  Alert,
+  FlatList,
+} from 'react-native';
+
+const ResultScreen = ({route, navigation}) => {
+
+    const [dataLoading, setDataLoading] = useState(true);
```

Comment:
Isso não vai causar nenhuma mudança no funcionamento, mas pra leitura do código é bacana quando variáveis boolenas tem nomes que parecem booleanos (perguntas de sim ou não, por exemplo). Acho que ao em vez de `dataLoading` seria bacana utilizar como `isDataLoading` ou  `isLoading`

Reply:
Complementando, se o nome da variável booleana for Value, os getters e setters são `isValue` e `setValue`